include/sundials/sundials_stepper.h

```diff
@@ -0,0 +1,125 @@
+/* -----------------------------------------------------------------
+ * SUNDIALS Copyright Start
+ * Copyright (c) 2002-2024, Lawrence Livermore National Security
+ * and Southern Methodist University.
+ * All rights reserved.
+ *
+ * See the top-level LICENSE and NOTICE files for details.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ * SUNDIALS Copyright End
+ * -----------------------------------------------------------------*/
+
+#ifndef _SUNDIALS_STEPPER_H
+#define _SUNDIALS_STEPPER_H
+
+#include <sundials/sundials_core.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* TryStep step result flags */
+#define SUNSTEPPER_TRYSTEP_FAILED  -1
+#define SUNSTEPPER_TRYSTEP_SUCCESS +0
+#define SUNSTEPPER_TRYSTEP_ADAPT   +1
```

Comment:
Can't we just forward flags from the inner method, e.g., `tmp_flag` in `arkStep_TryStep`? If so, then these are unnecessary.